felix/routerule/route_rule.go

```diff
@@ -144,6 +141,11 @@ func (r *RouteRules) getActiveRule(rule *Rule, f RulesMatchFunc) *Rule {
 
 // Set a Rule. Add to activeRules if it does not already exist based on matchForUpdate function.
 func (r *RouteRules) SetRule(rule *Rule) {
+
+	if r.netlinkFamily != rule.nlRule.Family {
+		log.WithField("rule", rule).Warnf("Rule does not match family %d, ignoring.", r.netlinkFamily)
```

Comment:
Are we hitting this anywhere?  Is logging sufficient - should we be ignoring the request?

Reply:
I do not think we are hitting this anywhere. Iirc it would be ignored later on and so it feels like why not to warn early. I think the best would be not to have Routes and RouteRules per family (internally it has to be), but that would be a bigger rework, save for another iteration :)